guides/v2.3/graphql/reference/products.md

```diff
@@ -167,6 +167,7 @@ Attribute | Data type | Description
 `categories` | [CategoryInterface] | The categories assigned to the product. See [categories endpoint]({{ page.baseurl }}/graphql/reference/categories.html) for more information
 `country_of_manufacture` | String | The product's country of origin
 `created_at` | String | Timestamp indicating when the product was created
+`crosssell_products` | [[ProductInterface](#ProductInterface)] | Cross-sell product(s)
```

Comment:
I would suggest clarifying the description slightly
```suggestion
`crosssell_products` | [[ProductInterface](#ProductInterface)] | An array of cross-sell products
```
This suggestion is fair for all types of related products

Reply:
**implemented**

Thanks, @rogyar